src/components/NoticeIcon/index.d.ts

```diff
@@ -3,6 +3,7 @@ import NoticeIconTab, { INoticeIconData } from './NoticeIconTab';
 
 export interface INoticeIconProps {
   count?: number;
+  bellIcon?: React.ReactNode;
```

Comment:
Is bellIcon inappropriate?
use NoticeIcon？

Reply:
I had a doubt if it should be called NoticeIcon, but I think it may cause conflict with the component if you have any suggestion on how to call it I can make a change.

Reply:
Bell has a clear meaning, it is a bell, but bellicon may be something else.

Reply:
so should I change it to 'bell' ?

Reply:
I changed the variable bellIcon to bell.